stratisd.sh

```diff
@@ -55,47 +55,3 @@ if [ -d $STRATIS_DEPS_DIR ]
 then
```

Comment:
I think it makes most sense to remove everything about ```STRATIS_DEPS_DIR``` since it is used for the stratis-cli deps only.